test/server.js

```diff
@@ -0,0 +1,100 @@
+const t = require('tap')
+const { serializeError } = require('../lib/error')
+const { redact, redactError, redactThrow } = require('../lib/server')
+const matchers = require('../lib/matchers')
+const examples = require('./fixtures/examples')
+
+t.test('redact', async t => {
+  await t.test('error has npm secret in message', async t => {
+    const error = new Error(`message contains npm secret: ${examples.NPM_SECRET.npm_36}`)
+    t.same(error.message, `message contains npm secret: ${examples.NPM_SECRET.npm_36}`)
+    const output = redact({ error })
+    t.same(output, {
+      error: {
+        errorType: 'Error',
+        message: `message contains npm secret: ${matchers.NPM_SECRET.replacement}`,
+        stack: error.stack.replace(examples.NPM_SECRET.npm_36, matchers.NPM_SECRET.replacement),
+      },
+    })
+  })
+  await t.test('error has npm secret in statusCode', async t => {
```

Comment:
If you add the equivalent of this test to `redactError` or `redactThrow` I think it will fail.

Reply:
I can add coverage for this, but this line in redact error should cover all the properties assigned to the error return `Object.assign(output, redact(data)` including status.

Im ok with serialize Error being in it's own file, it's not explicitly exported from the package anymore

Reply:
🤦🏼 I overlooked the call to `redact` on that line.